src/application.rs

```diff
@@ -45,19 +48,29 @@ impl<'a> Application<'a> {
         let mut event_queue = EventQueue::new();
 
         let sender = event_queue.sender().clone(); // Collect network events
-        let network = Network::new(move |net_event| sender.send(Event::Network(net_event)));
+        let network = Network::new(move |net_event| match net_event {
+            NetEvent::RemovedEndpoint(_) => sender.send_with_priority(Event::Network(net_event)),
+            _ => sender.send(Event::Network(net_event)),
+        });
 
         let sender = event_queue.sender().clone(); // Collect terminal events
         let _terminal_events = TerminalEventCollector::new(move |term_event| match term_event {
             Ok(event) => sender.send(Event::Terminal(event)),
             Err(e) => sender.send(Event::Close(Some(e))),
         })?;
+        #[allow(unused_mut)]
+        let mut commands = CommandManager::default().with(SendFileCommand);
+
+        #[cfg(target_os = "linux")]
```

Comment:
Could the static check be inside of the Send/StopStreamCommand? to leave this part clean? In the future who fixes it only need to touch the Command/Action related. Also, the default windows/mac implementation could be an error to the user :)

Reply:
Are you sure about this, instead of 3 cfg in total , I'll have to add multiple cfgs inside the send_stream file which is error prone

Reply:
I think currently its easy to reason whats gated behind an os flag, but if I put the gates inside send_stream file it wont be so obvious.

Reply:
Hmm I think I might be wrong I'll check it out

Reply:
Maybe is dirtier, but the dirty is located and isolated in the place related to it 💩.

> Are you sure about this, instead of 3 cfg in total , I'll have to add multiple cfgs inside the send_stream file which is error prone

What about making a `send_stream` folder with submodules for different OS implementations? Maybe is easy to make some "OS abstraction layer" that is what you use inside of `send_stream` module. In this way, you avoid using several cfgs.

Reply:
Another approach: Maybe the Action is the only part that should be dispatched by OS. You could only import the action used in your OS

Reply:
Also, this last approach allows you to be free inside of the action implementation to adapt to the OS-dependent streaming library. Each implementation attacks directly the native library, without an abstraction layer what makes it difficult

Reply:
Here is an attempt at this https://github.com/lemunozm/termchat/commit/1e423648adce0a9d2a3c15a99cc0db6fee6c1ff0

The problem with only modifying action and not command, is that we can't report an error that way if the user uses the command on an unsupported platform.

Reply:
I like how it looks! 👍🏻 

>The problem with only modifying action and not command, is that we can't report an error that way if the user uses the command on an unsupported platform.

I do not understand you well, It seems like the message`"{} command is not supported on this platform."` is reported as an error if the your are in a unsupported platform, right?

Reply:
Yes, I was talking about another approach (that keeps Command as a common struct, and just gate Action implementation).

It doesn't really matter I'll merge this then to the current PR